packages/salesforcedx-vscode-lwc/src/testSupport/testRunner/taskService.ts

```diff
@@ -126,7 +131,25 @@ class TaskService {
       sfdxTaskId: taskId
     };
     const taskSource = 'SFDX';
-    const taskShellExecution = new vscode.ShellExecution(cmd, args);
+    // https://github.com/forcedotcom/salesforcedx-vscode/issues/2097
+    // Git Bash shell doesn't handle command paths correctly.
+    // Always launch with command prompt (cmd.exe) in Windows.
+    const isWin32 = /^win32/.test(process.platform);
+    let taskShellExecutionOptions: vscode.ShellExecutionOptions | undefined;
+    if (isWin32) {
+      channelService.appendLine(
+        nls.localize('task_windows_command_prompt_messaging')
```

Comment:
Uses a generic message here - in case taskService could be reused by other extensions in the future.

Reply:
@mysticflute @lcampos Added the output panel message here